### PR TITLE
Update sql-block.md

### PR DESCRIPTION
**Pull request by zyw8136**
modify the document of the sql block rule,replace ":" with "="

# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by spaces-X — APPROVED** [review]
LGTM

**Review by spaces-X — APPROVED** [review]

File: docs/zh-CN/administrator-guide/block-rule/sql-block.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ under the License.
 CREATE SQL_BLOCK_RULE test_rule 
 PROPERTIES(
   "sql"="select * from order_analysis",
-  "global":"false",
+  "global"="false",
```
Comment (spaces-X) [inline docs/zh-CN/administrator-guide/block-rule/sql-block.md:52]: This pr is same to #8106

Comment (spaces-X) [reply docs/zh-CN/administrator-guide/block-rule/sql-block.md:52]: @zyw8136 pr #8106 has been approved. 
And this pr is repeated with #8106 maybe you can close this later.

Comment (zyw8136) [reply docs/zh-CN/administrator-guide/block-rule/sql-block.md:52]: o....
Others in the group got in first

Comment (zyw8136) [reply docs/zh-CN/administrator-guide/block-rule/sql-block.md:52]: @spaces-X  What do I need to do? Do I need to close this pr?

Comment (spaces-X) [reply docs/zh-CN/administrator-guide/block-rule/sql-block.md:52]: Yes, you can close this pr now.

**Comment by zyw8136** [comment]
It is same as the pr #8106

**Pull request closed without merging**